backend/de.metas.util/src/main/java/de/metas/util/ILoggable.java

```diff
@@ -36,8 +39,16 @@
 	 */
 	ILoggable addLog(String msg, Object... msgParameters);
 
-	/** Flush any buffered logs */
+	/**
+	 * Flush any buffered logs
+	 */
 	default void flush()
 	{
 	}
+
+	default ILoggable addTableRecordReferenceLog(final ITableRecordReference recordRef, final String type, final String trxName)
+	{
+		return this;
```

Comment:
please "render" a log message string and call `addLog()` with it

Reply:
Sure